Project.toml

```diff
@@ -3,7 +3,27 @@ uuid = "0bca4576-84f4-4d90-8ffe-ffa030f20462"
 authors = ["Chris Rackauckas <accounts@chrisrackauckas.com> and contributors"]
 version = "0.1.0"
 
+[deps]
+CommonSolve = "38540f10-b2f7-11e9-35d8-d573e4eb0ff2"
+DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
+LabelledArrays = "2ee39098-c373-598a-b85f-a56591580800"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
+RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
+RecursiveArrayTools = "731186ca-8d62-57ce-b412-fbd966d074cd"
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
+TreeViews = "a2a6695c-b41b-5b7d-aed9-dbfdeacea5d7"
```

Comment:
I was a bit surprised to see that the latest release is from July 2018 and the repo contains no Project.toml but only a REQUIRE file. But I assume it just indicates that the package is very stable and no changes are expected?

Reply:
It's super stable. @pfizseb

Reply:
Yep. I don't think there are any actively maintained consumers of TreeViews's tree views anymore though ;)

Reply:
What do we define for VSCode and Juno nowadays then? TreeViews still seems to work for Juno?

Reply:
Yes. VSCode's tree views (in the workspace) aren't customizable atm.